packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.tsx

```diff
@@ -51,6 +52,15 @@ export const UnitRowTwoStepAuth = () => {
     }
   }, [account, hideModal, alertBar, l10n]);
 
+  console.log('Number of backup codes remaining: ', count);
+
+  // TODO in FXA-10206, remove this console log and use the count data for the backup codes subrow
```

Comment:
This console log is temporary and can be used to test that the number of remaining backup authentication codes is retrieved